frontend/src/routes/HomePage.js

```diff
@@ -270,7 +270,7 @@ class HomePage extends React.Component<{
                                         {this.renderTableItem(
                                             'REST API Reference',
                                             'Develop your own application against our REST API.',
```

Comment:
As a suggestion, can we move these strings to separate JSON files rather than directly hardcoding them in JS? It would be useful and easy to localize them if we decide to support multiple languages later on.

Reply:
Yeah, do you want to make an issue for that so we could address it later?